backend/server/manage.py

```diff
@@ -1,14 +1,27 @@
+from flask_script import Command, Manager, Shell
+
 from oceanhub import app
 from oceanhub.scripts.shell import make_shell_context
-from flask_script import Manager, Command, Shell
 
 manager = Manager(app)
 
+
 class StartShell(Command):
     def run(self):
         Shell(make_context=make_shell_context).run(False, False, False, False)
 
-manager.add_command("shell", StartShell())
 
-if __name__ == "__main__":
+class RunServer(Command):
+    def run(self):
+        app.run(
+            host='0.0.0.0',
+            debug=True,
+            port=5000
+        )
+
+
+manager.add_command('shell', StartShell())
```

Comment:
If we are using the Flask CLI, do we really need this manager?

Reply:
Sorry for going backward, approving, then commenting haha. My brain churned further :confused:

Reply:
hey we are using flask-script

Reply:
yea was asking if we are using the Flask CLI :stuck_out_tongue: